draft-ietf-quic-ack-frequency.md

```diff
@@ -251,25 +251,25 @@ Request Max Ack Delay:
 
 Reordering Threshold:
 
-: A variable-length integer that indicates how many
+: A variable-length integer that indicates how
   out of order packets can arrive before eliciting an immediate ACK. If no
   ACK_FREQUENCY frames have been received, the endpoint immediately acknowledges
   any subsequent packets that are received out of order, as specified in
-  {{Section 13.2 of QUIC-TRANSPORT}}, as such the default value is 1.
+  {{Section 13.2 of QUIC-TRANSPORT}}, corresponding to a default value of 1.
   A value of 0 indicates out-of-order packets do not elicit an immediate ACKs.
 
 ACK_FREQUENCY frames are ack-eliciting. When an ACK_FREQUENCY frame is lost,
-is encouraged to send an ACK_FREQUENCY frame, unless an ACK_FREQUENCY frame
-with a larger Sequence Number value has already been sent. However, it is not
-forbidden to retransmit the lost frame (see Section 13.3 of {{QUIC-TRANSPORT}),
+it is recommenended to send another ACK_FREQUENCY frame, unless an ACK_FREQUENCY
```

Comment:
"recommended" is also a RFC2119 keyword which I think we tried to avoid.

Reply:
Agreed.  Changed back to encouraged.